### DISCUSSION TITLE
Trivy should be deployed in Server/Client mode in the helm chart

### DISCUSSION THREAD

**Discussion by MarkusTeufelberger** [aquasecurity/starboard#886] [Development]
Without a Github token Trivy quickly runs into rate limiting issues - and even when adding a token this seems like a waste of resources to download everything over and over again. There should at least be an option to automatically deploy + configure Trivy in Server/Client mode instead of leaving this up to the user to implement.

Helm charts can also reference other charts and Trivy already has a chart available too, so there should not be too much friction here. The Trivy chart might need some love though (e.g. authentication, HTTPS options...), the starboard-operator chart expects a more fully featured deployment than the Trivy chart currently offers (which references an outdated version too at the moment).

**Comment by krol3** [comment]
Hi @MarkusTeufelberger, I updated the trivy chart app version in this PR: https://github.com/aquasecurity/trivy/pull/1535.
Please share with us, other improvements.
